src/open-data/Services.csv

```diff
@@ -1,314 +1,314 @@
-ConsumedService,ResourceType,ServiceName,ServiceCategory,PublisherName,PublisherType
```

Comment:
Can you run `scripts/Build-OpenData.ps1 "Services"` to update the PowerShell command and then run `scripts/Test-PowerShell.ps1 -OpenData` to make sure everything is good?

Reply:
You might want to update your branch. I made a change to that script recently which will make sure you don't update a bunch of stuff that doesn't need to be updated.